include/BlitzHelpers.hpp

```diff
@@ -167,4 +167,28 @@ namespace blitzdg {
 	void fullToVector(const matrix_type<T>& mat, vector_type<T>& vec, bool byRows = true) {
 		reshapeMatTo1D(mat, vec.begin(), byRows);
 	}
+
+    /**
+     *  Computes Kronecker product of matrices A and B.
+     *  @param[in] A The left matrix.
+     *  @param[in] B The right matrix.
+     *  @returns C = kron(A,B)
+     */
+    template <typename T>
+    matrix_type<T> kron(const matrix_type<T>& A, const matrix_type<T>&B) {
```

Comment:
This looks good buddy, but I wonder if it could be made a bit cleaner by declaring variables for `B.rows()` and `B.cols()`. You could do something like:
```
using range = blitz::Range;
const index_type Br = B.rows(), Bc = B.cols();
for (index_type i = 0; i < A.rows(); ++i) {
    for (index_type j = 0; j < A.cols(); ++j) {
        C(range(j * Br, (j + 1) * Br - 1), 
          range(i * Bc, (i + 1) * Bc - 1)) = A(i,j) * B;
    }
}   
```
Naturally you could move the indexing for the column extents to the outer loop by declaring a new variable there, but the compiler probably already does this for you.